config/crds/hive.openshift.io_clusterdeployments.yaml

```diff
@@ -617,6 +642,142 @@ spec:
                         TODO: Add other useful fields. apiVersion, kind, uid?'
                       type: string
                   type: object
+                installStrategy:
+                  description: InstallStrategy provides platform agnostic configuration
+                    for the use of alternate install strategies. Defaults to openshift-install
+                    if none specified.
```

Comment:
How should we conceptualize the openshift-install InstallStrategy?

e.g. it means "clusters created by running `openshift-install create cluster`" ?

So a future installStrategy.openshiftInstall field would contain stuff only relevant to `openshift-install create cluster` type deployments?

Reply:
Yes that is what we had in mind I believe, essentially control over the implementation used to deploy the cluster, and settings specific to just one stategy would nest within.